backend/cdb_csv/pe.py

```diff
@@ -93,7 +96,7 @@ async def match_beneficiaries_and_pros(connection: Connection, principal_csv: st
             if beneficiary and beneficiary.notebook is not None:
 
                 logging.info(
-                    f"{pe_unique_id} - Found matching beneficiary {beneficiary.id}"
+                    f"{pe_unique_id} - Found matching beneficiary for pro {beneficiary.id}"
```

Comment:
pourquoi tu as rajouté `for pro` sachant que tu affiche un `beneficiairy_id` ?

Reply:
Parce que c'est dans le partie qui matche les benef et les pros (le hot fix)